docs/image.rst

```diff
@@ -0,0 +1,108 @@
+Image
+*****
+
+.. py:module:: microbit
+
+Classes
+=======
+
+.. py:class:: Image(width_or_string=None, height=None, buffer=None)
+
+    If ``width_or_string`` is a string, creates an image from that string.
+
+    If it's an integer, a height has to also be specified, and it creates
+    an image with ``width_or_string`` columns and ``height`` rows. Optionally
+    ``buffer`` can be an array of ``width_or_string``×``height`` integers to
+    initialize the image.
```

Comment:
IMO, the string constructor is the "natural" one. If we were targeting adult developers, then I would like to turn the buffer constructor into a `from_buffer` static method.

Also, given that brightnesses are in `range(10)`, the buffer constructor has a rather odd interface.
Ideally I would like `Image.bw_from_data(width, height, data)` and `Image.grey_from_data(width, height, data)`
To create a 2x2 black and white image, one would write `Image.bw_from_data(2, 2, 0b0101)`
To create a 2x2 greyscale image, one would write `Image.grey_from_data(2, 2, 0909)`